content-plugins.md

```diff
@@ -131,18 +119,17 @@ The initial release contains the following functionality:
 
 * Use Pulp as a pull through cache for Maven content.
 
-
-### Unmaintained plugins
-
-The following plugins have been created and developed to some extent but have not been regularly maintained.
-If you're interested in extending or maintaining the basic functionality of these plugins, [let us know](/get_involved/).
-
 #### Ruby Gem
 
 * Synchronize remote repository content and metadata locally.
 * Upload your own content.
 * Publish content to one or more repositories.
 
+### Unmaintained plugins
+
+The following plugins have been created and developed to some extent but have not been regularly maintained.
+If you're interested in extending or maintaining the basic functionality of these plugins, [let us know](/get_involved/).
```

Comment:
```suggestion
If you're interested in extending or maintaining the basic functionality of these plugins, [let us know](/help/).
```

get_involved is gone now, yes? Is there a particular anchor (`## Contribute as a Developer` maybe?) Or should this point to `[developers guide](https://docs.pulpproject.org/contributing/index.html)` here?

Reply:
yes, you're right it should be `help`